src/Drupal/ModuleExtension.php

```diff
@@ -0,0 +1,55 @@
+<?php declare(strict_types=1);
+
+namespace mglaman\PHPStanDrupal\Drupal;
+
+final class ModuleExtension
```

Comment:
I think it'd be good to just add these details to the existing `Extension` object. Originally the idea was to keep a more "pure" fork of the Drupal code... but I have given up on that :)

Reply:
I think we should try to have different objects because modules, themes and profiles have different domain behaviours.

They have different configurations and informations inside there `.info.yml`.

For instance themes have specific parameters like `regions`, `libraries`, `engine` whereas profiles have `install`, `distribution` and modules have `test_dependencies` etc..

It will be simple to work and avoid a god object. WDYT?

Reply:
Will we be inspecting the internal properties, though? I guess down the line. But right now the main thing is for properties that are the same across any extension type.